apps/payments/next/app/lib/stubs.ts

```diff
@@ -0,0 +1,115 @@
+/**
+ *
+ * Temporary File
+ *
+ * This is a temporary file holding stub async calls
+ *
+ * Currently this file stubs the following queries
+ * - Cart Data fetch
+ * - Contentful data fetch by offering
+ *
+ */
+
+export interface PurchaseDetails {
+  details: string[];
+  productName: string;
+  subtitle: string;
```

Comment:
As discussed, this should be `subtitle?` as subtitle is not required in Purchase Details per Technical Specification.

Reply:
As discussed, I've fixed this in the PurchaseDetails component, `libs/payments/ui/src/lib/server/purchase-details.tsx`, as well.

As you mentioned, this file is a stub and will be removed with actual implementation. However the `PurchaseDetails` component is the actual implementation that will remain once the stubs are replaced with actual implementations.